sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/BlockBlobAPITest.groovy

```diff
@@ -732,7 +750,9 @@ class BlockBlobAPITest extends APISpec {
         thrown(StorageException)
     }
 
+    // Only run these tests in live mode as they use variables that can't be captured.
     @Unroll
+    @Requires({ APISpec.liveMode() })
```

Comment:
Can we split two two different tests? There are bunch of small data which we are able to test in playback.

Reply:
The issue isn't the data size but the fact that there are variables that get used inside of the method call that can't be captured by the playback system.

Reply:
Why we cannot record the variables?